### PR TITLE
[lldp] fix issue #79 - AttributeError: 'NoneType' object has no attribute 'decode'

### LINKED ISSUE #79: AttributeError: 'NoneType' object has no attribute 'decode'
In the latest master image, the newly pulled in commit: https://github.com/Azure/sonic-snmpagent/pull/72 will cause issue when the information cannot be retrieved from the database.

`Aug  9 23:29:26.815914 SONiC ERR snmp/snmp-subagent [ax_interface] ERROR: MIBUpdater.start() caught an unexpected exception during update_data()#012Traceback (most recent call last):#012  File "/usr/local/lib/python3.6/dist-packages/ax_interface/mib.py", line 40, in start#012    self.reinit_data()#012  File "/usr/local/lib/python3.6/dist-packages/sonic_ax_impl/mibs/ieee802_1ab.py", line 270, in reinit_data#012    b'lldp_loc_man_addr').decode('utf-8')#012AttributeError: 'NoneType' object has no attribute 'decode'`

### PR DESCRIPTION
**Pull request by mykolaf**
fixes #79 
**- What I did**
Added handling of case when some "LOC_CHASSIS" data is absent in APP DB.
**- How I did it**
Added a check if we received something from the DB.
**- How to verify it**
**- Description for the changelog**
Fix snmpagent issue: AttributeError: 'NoneType' object has no attribute 'decode'

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by qiluo-msft** [comment]
retest this please

File: src/sonic_ax_impl/mibs/ieee802_1ab.py [hunk 1]
```diff
@@ -266,8 +266,13 @@ def reinit_data(self):
...
         self.db_conn.connect(mibs.APPL_DB)
-        self.mgmt_ip_str = self.db_conn.get(mibs.APPL_DB, mibs.LOC_CHASSIS_TABLE,
-                                            b'lldp_loc_man_addr').decode('utf-8')
+        mgmt_ip_bytes = self.db_conn.get(mibs.APPL_DB, mibs.LOC_CHASSIS_TABLE, b'lldp_loc_man_addr')
+
+        if not mgmt_ip_bytes:
+            logger.warning("failed to get local management address from APP DB.")
+            return
+
+        self.mgmt_ip_str = mgmt_ip_bytes.decode('utf-8')
```
Comment (qiluo-msft) [inline src/sonic_ax_impl/mibs/ieee802_1ab.py]: >'utf-8' [](start = 48, length = 7)

this is the default, so you can safely remove it.

**Pull request merged**